index.html

```diff
@@ -320,6 +320,12 @@ <h2 class="contributors-head" id="contributer">Contributors</h2>
         <div class="well contributors"><div id="spinner-icon" style="text-align: center;"><i class="fa fa-spinner fa-spin"></i></div><p class="usernames"></p><hr /><p class="avatars text-center"></p></div>
       </div>
 
+      <!-- Dependency chart  -->
+      <div id="dependency-div">
+      <img  class="dependency-chart" src="./images/chart.png" alt="dependency-chart">
```

Comment:
Wait, is the chart already in our repository, or do we need to add that file too? Thank you!!!

Reply:
Yes, @jywarren I have added that file too.
Warm Regards,
Yogesh Sharma
Thanks